xpdan/formatters.py

```diff
@@ -65,11 +66,48 @@ def clean_template(template, removals=None, cfmt=cfmt):
     e = e.replace(']', '')
     e = e.replace('(', '')
     e = e.replace(')', '')
+    e = e.replace('\'','')
     f = Path(e).as_posix()
     f = f.replace('/_', '/')
     return f
 
+# Replaces each parameter name within folder_tag_list with the value of that parameter.
+def get_filename_prefix(folder_tag_list,md):
+    result =''
+    for tag in folder_tag_list:
+        if isinstance(tag,str):
+            try:
+                raw_addition = md[tag]
```

Comment:
you might want to use the `dict.get(key, default)` syntax so you won't need the try except.
```python
raw_addition = md.get(tag, '')
```